libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/Zigbee_Light_Bulb.ino

```diff
@@ -0,0 +1,193 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @brief This example demonstrates simple Zigbee light bulb.
+ * 
+ * The example demonstrates how to use ESP Zigbee stack to create a end device light bulb.
+ * The light bulb is a Zigbee end device, which is controlled by a Zigbee coordinator.
+ * 
+ * Please check the README.md for more detailed description.
+ */
+
+#ifndef ZIGBEE_MODE_ED
+#error "Zigbee end device mode is not selected in Tools->Zigbee mode"
+#endif
+
+#include "esp_zigbee_core.h"
+#include "nvs_flash.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "ha/esp_zigbee_ha_standard.h"
+
+#define LED_PIN RGB_BUILTIN
+
+/* Default End Device config */
+#define ESP_ZB_ZED_CONFIG()                                     \
+    {                                                           \
+        .esp_zb_role = ESP_ZB_DEVICE_TYPE_ED,                   \
+        .install_code_policy = INSTALLCODE_POLICY_ENABLE,       \
+        .nwk_cfg = {                                            \
+            .zed_cfg = {                                        \
+                .ed_timeout = ED_AGING_TIMEOUT,                 \
+                .keep_alive = ED_KEEP_ALIVE,                    \
+            },                                                  \
+        },                                                      \
+    }
+
+#define ESP_ZB_DEFAULT_RADIO_CONFIG()                           \
+    {                                                           \
+        .radio_mode = RADIO_MODE_NATIVE,                        \
+    }
+
+#define ESP_ZB_DEFAULT_HOST_CONFIG()                            \
+    {                                                           \
+        .host_connection_mode = HOST_CONNECTION_MODE_NONE,      \
+    }
+
+/* Zigbee configuration */
+#define INSTALLCODE_POLICY_ENABLE       false    /* enable the install code policy for security */
+#define ED_AGING_TIMEOUT                ESP_ZB_ED_AGING_TIMEOUT_64MIN
+#define ED_KEEP_ALIVE                   3000    /* 3000 millisecond */
+#define HA_ESP_LIGHT_ENDPOINT           10    /* esp light bulb device endpoint, used to process light controlling commands */
+#define ESP_ZB_PRIMARY_CHANNEL_MASK     ESP_ZB_TRANSCEIVER_ALL_CHANNELS_MASK  /* Zigbee primary channel mask use in the example */
+
+/********************* Zigbee functions **************************/
+static void bdb_start_top_level_commissioning_cb(uint8_t mode_mask)
+{
+    ESP_ERROR_CHECK(esp_zb_bdb_start_top_level_commissioning(mode_mask));
+}
+
+void esp_zb_app_signal_handler(esp_zb_app_signal_t *signal_struct)
+{
+    uint32_t *p_sg_p       = signal_struct->p_app_signal;
+    esp_err_t err_status = signal_struct->esp_err_status;
+    esp_zb_app_signal_type_t sig_type = (esp_zb_app_signal_type_t)*p_sg_p;
+    switch (sig_type) {
+    case ESP_ZB_ZDO_SIGNAL_SKIP_STARTUP:
```

Comment:
`ESP_ZB_ZDO_SIGNAL_SKIP_STARTUP` is IDF in the sense that, for instance, in WiFi, we use clear and Arduino names. Shouldn't we do the same for Arduino Zigbee?